cassandra-migration/src/test/java/org/cognitor/cassandra/migration/scanner/JarLocationScannerTest.java

```diff
@@ -28,6 +31,11 @@ public class JarLocationScannerTest {
     private static File jarFile;
     private static URI jarUri;
 
+    @Before
+    public void windowsOnly() {
+        Assume.assumeFalse(isWindows());
```

Comment:
I think it makes sense to provide a message here. Just failing without explanation is kind of rude :)

Reply:
Thanks for the suggestion. One thing is it doesn't fail, it just doesn't run tests if the Assume is not satisfied.
It's kind of a conditional @Ignore ;)

Reply:
I see. Thanks for explaining :)